.gitmodules

```diff
@@ -12,3 +12,6 @@
 [submodule "lib/forge-std"]
 	path = lib/forge-std
 	url = https://github.com/foundry-rs/forge-std
+[submodule "lib/@matterlabs/zksync-contracts"]
```

Comment:
Why do we need this submodule? Looks good otherwise

Reply:
We need this library for efficientcall in the l2-contracts.

We try to avoid importing stuff from `node_modules` inside foundry, so we have to import it via submodule like a normal foundry lib